vector/src/main/java/im/vector/app/features/home/HomeActivityViewState.kt

```diff
@@ -23,5 +23,4 @@ import org.matrix.android.sdk.api.session.sync.SyncRequestState
 data class HomeActivityViewState(
         val syncRequestState: SyncRequestState = SyncRequestState.Idle,
         val authenticationDescription: AuthenticationDescription? = null,
-        val areNotificationsSilenced: Boolean = false,
```

Comment:
This is (or will be) handled elsewhere?

Reply:
I need to look into that, I am not sure it is needed. What is sure is that it should not have been addedd in the `ViewState` as it was not used for rendering UI but for domain logic. There will be 2 successive PRs following that current PR which will fix issues with both remote and local notifications toggling for the session.

Reply:
OK, thanks!